operators/operator_sdk/java/osdk-java-quickstart.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="osdk-java-quickstart"]
+= Getting started with Operator SDK for Java-based Operators
+include::_attributes/common-attributes.adoc[]
+:context: osdk-java-quickstart
+:FeatureName: Java-based Operator SDK
+include::snippets/technology-preview.adoc[]
+
+toc::[]
+
+To demonstrate the basics of setting up and running a Java-based Operator using tools and libraries provided by the Operator SDK, Operator developers can build an example Java-based Operator for Memcached, a distributed key-value store, and deploy it to a cluster.
+
+include::modules/osdk-common-prereqs.adoc[leveloffset=+1]
+include::modules/osdk-quickstart.adoc[leveloffset=+1]
+
+[id="osdk-java-quickstart-next-steps"]
```

Comment:
Nit-picky, but per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies):

```suggestion
[id="next-steps_osdk-java-quickstart"]
```